### PR TITLE
[CSM-334] createTransaction takes a long time

### PR DESCRIPTION
**Pull request by pva701**
* Introduce local utxo: store map of wallet's addresses in wallet-db. Got rid of iterating over whole GState Utxo in `getOwnUtxos`

### CODE REVIEW

**Comment by akegalj** [comment]
I have tested it and confirm that it works https://issues.serokell.io/issue/CSM-334#comment=96-5205 . I have also reviewed the code and it looks ok to me. Would like @flyingleafe to review the code as well before we merge

**Comment by akegalj** [comment]
NOTE: don't merge yet - there are possibly some problems found https://issues.serokell.io/issue/CSM-334#comment=96-5208 . They might be related to CSM-372 https://issues.serokell.io/issue/CSM-327 though - testing in progress

File: src/Pos/Client/Txp/Balances.hs [hunk 1]
```diff
@@ -66,18 +67,18 @@ runBalancesRedirect :: BalancesRedirect m a -> m a
 runBalancesRedirect = coerce
 
 instance
-    (MonadRealDB m, MonadDBRead m, MonadMask m, WithLogger m, MonadTxpMem ext m, t ~ IdentityT) =>
+    ( MonadDBRead m
+    , WebWalletModeDB m
+    ,  MonadMask m
```
Comment (flyingleafe) [inline src/Pos/Client/Txp/Balances.hs]: uberi probel 🔪

**Comment by gromakovsky** [comment]
All commits have wrong issue id. It's really bad 🔪

**Pull request merged**